main/src/main/scala/sbt/internal/LibraryManagement.scala

```diff
@@ -332,7 +329,7 @@ private[sbt] object LibraryManagement {
         import Keys._
         val cacheDirectory = s.cacheDirectory
         val isRoot = er.contains(rs)
-        if csr then {
+        if true then {
```

Comment:
I'm assuming you're doing this so that you can keep the (now dead) ivy code around. Doesn't this produce a compiler warning about an unreachable code path in the `else`?

I feel like version control protects us here, and we can just remove the dead code. Or if you want to feel safer, do what you've done in other places - comment out the `if`, and the whole `else` block.

Reply:
Addressed in 80fabeeb2c5780541f0900100aeef1e02dfa3b78